k8s-daemonset/k8s/linkerd-namerd.yml

```diff
@@ -0,0 +1,130 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: l5d-config
+data:
+  config.yaml: |-
+    admin:
+      port: 9990
+
+    namers:
+    - kind: io.l5d.k8s
+      experimental: true
+      host: localhost
+      port: 8001
+
+    routers:
+    - protocol: http
+      label: outgoing
+      interpreter:
+        kind: io.l5d.namerd
+        dst: /$/inet/namerd.default.svc.cluster.local/4100
+        namespace: outgoing
+        transformers:
+        - kind: io.l5d.k8s.daemonset
+          namespace: default
+          port: incoming
+          service: l5d
+      servers:
+      - port: 4140
+        ip: 0.0.0.0
+
+    - protocol: http
+      label: incoming
+      dstPrefix: /
+      identifier:
+        kind: io.l5d.header
+        header: l5d-dst-concrete
+      interpreter:
+        kind: default
```

Comment:
Could use the namerd interpreter on the incoming router as well to avoid needing to define the k8s namer above.